src/nvim/buffer.c

```diff
@@ -2629,7 +2629,7 @@ void maketitle(void)
 
   if (p_title) {
     if (p_titlelen > 0) {
-      maxlen = p_titlelen * Columns / 100;
+      maxlen = (int)(p_titlelen * Columns) / 100;
```

Comment:
why not `(int)(p_titlelen * Columns / 100)` as it is the type of the end result that matters? (I know 32-bit division is faster, but this is hardly perfomance critical section)